prov/verbs/src/verbs_cq.c

```diff
@@ -36,6 +36,7 @@
 
 #include "fi_verbs.h"
 
+static uint64_t fi_ibv_comp_flags(struct ibv_wc *wc);
```

Comment:
Thank you for the fix. The function can be moved here to avoid adding this declaration.

Btw, why is the signed-off line in the commit message has Sean's name in it?